.github/workflows/projects_check.yml

```diff
@@ -8,18 +8,29 @@ on:
     paths:
     - '.github/workflows/xcode_check.yml'
     - 'EndlessSky.xcodeproj/**'
+    - 'EndlessSkyLib.cbp'
+    - 'EndlessSkyTests.cbp'
     - 'source/**'
     - 'tests/check_xcode.sh'
+    - 'tests/check_codeblocks.sh'
   pull_request:
     types: [opened, synchronize]
     paths:
     - '.github/workflows/xcode_check.yml'
     - 'EndlessSky.xcodeproj/**'
+    - 'EndlessSkyLib.cbp'
+    - 'EndlessSkyTests.cbp'
     - 'source/**'
     - 'tests/check_xcode.sh'
+    - 'tests/check_codeblocks.sh'
```

Comment:
```suggestion
```
Let's run this workflow file on every push to any branch

Reply:
The workflow doesn't run on PRs anymore due to this change I think.

Reply:
Every other workflow runs on specific branches. Why not here?